### ISSUE TITLE
Provide dockerpty.__version__ 

### ISSUE THREAD

**Issue by yarikoptic** [d11wtq/dockerpty#75]
I guess ATM should be assigned  `"0.4.1"`
Unfortunately there is no canonical way to setup version to be maintained in a single location but used within module and setup.py (although there are a bunch of `versioneer`s projects), so please choose your favorite way ;)  I found simplest is to have `module/version.py` to be imported into `__init__.py` and either parse or exec it.  if you would like -- can do it that way for dockerpty

**Comment by graingert** [comment]
@yarikoptic \_\_version\_\_ is redundant. Use pkg_resources.get_distribution('dockerpty').version

**Comment by yarikoptic** [comment]
good point!  So is it just historically happened that majority of modules provide it ?  or may be to provide version information for naive / not-installed deployments where you merely point PYTHONPATH to the location of the module (hence no pkg_resources info)?

**Comment by graingert** [comment]
Yeah \_\_version\_\_ predates pkg_resources and packaging being good
